service/controller/resource/instance/template/main.json

```diff
@@ -262,7 +265,7 @@
               "tags":{
                 "provider":"[toUpper(parameters('GiantSwarmTags').provider)]",
                 "cluster-autoscaler-enabled":"true",
-                "cluster-autoscaler-name":"[parameters('vmssName')]",
+                "cluster-autoscaler-name":"[parameters('clusterID')]",
```

Comment:
the cluster autoscaler app uses just the cluster ID as name

Reply:
See https://github.com/giantswarm/cluster-autoscaler-app/blob/master/helm/cluster-autoscaler-app/templates/deployment.yaml#L52